locale.c

```diff
@@ -8629,7 +8629,10 @@ S_strftime8(pTHX_ const char * fmt,
 
       case UTF8NESS_YES:    /* Known to be UTF-8; must be UTF-8 locale if can't
                                downgrade. */
-        if (! is_locale_utf8(locale)) {
+        if (is_locale_utf8(locale)) {
+            locale_utf8ness = LOCALE_IS_UTF8;
+        }
+        else {
```

Comment:
maybe drop the curlys for less whitespace/more code on 1 screen for the `if()`?

Its upto you, I dont have much of an opinion either way.

`LOCALE_IS_UTF8` is a const from an `enum` according to grep, not a `STMT_START {` thing or a complicated long expression so I dont see a real or paranoia precedence/syntax error hazard here for inserting `{}` for a 1 liner.

Reply:
I have been burned too many times by adding, say, a debugging statement that broke things because I forgot to add braces at the same time.